django_mongodb_backend/routers.py

```diff
@@ -16,3 +16,29 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
         except LookupError:
             return None
         return False if issubclass(model, EmbeddedModel) else None
+
+
+class EncryptionRouter:
+    """
+    Routes database operations for 'encrypted' models to the 'encryption' DB.
+    """
+
+    def db_for_read(self, model, **hints):
+        if getattr(model, "encrypted_fields_map", False):
+            return "encryption"
+        return None
+
+    def db_for_write(self, model, **hints):
+        if getattr(model, "encrypted_fields_map", False):
+            return "encryption"
+        return None
+
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
+        """
+        Ensure that the 'encrypted' models only appear in the 'encryption' DB,
+        and not in the default DB.
+        """
+        model = hints.get("model")
+        if model and getattr(model, "encrypted_fields_map", False):
+            return db == "encryption"
+        return None
```

Comment:
I think we should provide an example in the documentation but probably not here as a public API since making a sufficiently generic router seems questionable (for example, hardcoding "encryption" as the name of the alias). I'd suggest putting this code in `tests/encryption_/routers.py` and using it there with `@override_settings(DATABASE_ROUTERS=...`. (And as discussed in the design doc, stop using `encrypted_fields_map` to detect encrypted models. At the least, adding a `encrypted = True` class attribute on `EncryptedModel` would be consistent with `EncryptedField` and wouldn't be so ugly until we decide on a final solution.)